.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,34 @@
+<!--- 
 Thanks for contributing to manim!
-
 **Please ensure that your pull request works with the latest version of manim.**
```

Comment:
```suggestion
**Please ensure that your pull request works with the latest version of manim from this repository.**
```
Clarification: Might want to mention that we are talking about the ManimCommunity version of Manim here.

Reply:
Committed.